local-cli/templates/HelloWorld/android/build.gradle

```diff
@@ -36,10 +36,10 @@ ext {
     minSdkVersion = 16
     compileSdkVersion = 26
     targetSdkVersion = 26
-    supportLibVersion = "26.1.0"
+    supportLibVersion = "27.1.1"
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '4.4'
+    gradleVersion = '4.8'
```

Comment:
why 4.8 here ?

Reply:
also use wrapper task to update, don't change gradle-wrapper.properties directly.